iOS Client/Store/MBStore.swift

```diff
@@ -22,6 +22,21 @@ class MBStore: NSObject {
         super.init()
     }
     
+    
+    static func loadState() -> MBAppState {
+        let defaults = UserDefaults.standard
+        var notificationPermission: Permission = .unprompted
+        if let notificationSetting = defaults.object(forKey: "notifications") as? Bool {
+            notificationPermission = notificationSetting ? .approved : .denied
+        }
+        return MBAppState(navigationState: MBNavigationState(), articleState: MBArticleState(), devotionState: MBDevotionState(), settingsState: MBSettingsState(notificationPermission: notificationPermission))
+    }
+    
+    static func saveState(state: AppState) {
```

Comment:
where are these functions being called?

Reply:
These were just an idea I had to save and load the state. I didn't actually get around to using them so I will remove for now